ui/proposal_details_vote_modal.go

```diff
@@ -0,0 +1,260 @@
+package ui
+
+import (
+	"fmt"
+	"image/color"
+	"strconv"
+
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/widget"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const ModalInputVote = "input_vote_modal"
+
+type inputVoteOptionsWidgets struct {
+	label      string
+	background color.NRGBA
+	input      decredmaterial.Editor
+	inc        decredmaterial.IconButton
+	dec        decredmaterial.IconButton
+	max        decredmaterial.Button
+}
+
+type inputVoteModal struct {
```

Comment:
`voteModal` is just fine for the struct name. The file name can also be vote_modal or proposal_vote_modal.